cleverhans/utils_th.py

```diff
@@ -62,6 +62,39 @@ def adadelta(loss_or_grads, params, learning_rate=1.0, rho=0.95, epsilon=1e-6):
 
     return updates
 
+def adam(loss_or_grads, params, learning_rate=0.001, beta1=0.9,
```

Comment:
well, i think that one should also be removed, maybe similar to the way [it's done in JSMA](https://github.com/openai/cleverhans/blob/master/cleverhans/attacks_tf.py#L148-L152). Or we can think of a more robust solution